### PR TITLE
:memo: add README info

### PR DESCRIPTION
**Pull request by joerunde**
Adds some info to the readme about usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,42 @@
...
-models for delivery both internal and external to IBM.
+models for delivery with an embeddable watson runtime.
+
+## Overview
+
+Given a set of `watson_embed` models to be packaged, this tool can create a model manifest file describing the model
+images and metadata to be created.
+
+Given a model manifest file, this tool can fetch the model artifacts and package them into watson runtime compatible 
+model images alog with useful metadata.
```
Comment (aluu317) [inline README.md:12]: along

Comment (joerunde) [reply README.md:12]: it's an inside joke

File: README.md [hunk 2]
```diff
@@ -1,4 +1,48 @@
...
+
+```shell
+pip install watson_embed_model_packager
+```
+
+## Usage
+
+To build a model manifest file from models hosted in an artifactory instance, you will need:
+
+- A list of all module GUIDs to support
```
Comment (aluu317) [inline README.md:26]: nit: this list has 2 capitalized letters, and 3 non-capitalized letters lol. Let's pick one of the other!

**Review by aluu317 — APPROVED** [review]
Yas!

**Pull request merged**